assets/src/styles/blocks/OldENForm/components/_enform.scss

```diff
@@ -90,7 +90,7 @@
   }
 }
 
-.form-description p:first-child --block-enform--caption--paragraph--first-- {
+.form-description p:first-child _-- {
```

Comment:
Changing this back to short notation as the only purpose is to ensure continuity for the "climate" theme. We can remove this rule afterwards, as what it's effectively doing is turning the first paragraph into a title, while the block also has a title.